### DISCUSSION TITLE
What's the difference between the 'full' and not 'full' version in linux amd64?

### DISCUSSION THREAD

**Discussion by eliphatfs** [seaweedfs/seaweedfs#5309] [General]
This sounds stupid but I didn't find it in the documentation and I cannot 'get started' because I do not know what to download.
Seems that both are working on my local machine, but I am curious about it.

**Comment by chrislusf** [comment]
full has other support for `elastic,gocdk,sqlite,ydb,tikv,rclone`, which was skipped in normal version to reduce binary size.
